packages/common/container-definitions/src/error.ts

```diff
@@ -3,72 +3,32 @@
  * Licensed under the MIT License.
  */
 
-import { ITelemetryProperties } from "@fluidframework/core-interfaces";
+import { IErrorBase } from "@fluidframework/core-interfaces";
 
 /**
- * Different error types the Container may report out to the Host
+ * @deprecated IErrorBase is being deprecated as a public export is moving to "core-interfaces".
  */
-export enum ContainerErrorType {
-	/**
-	 * Some error, most likely an exception caught by runtime and propagated to container as critical error
-	 */
-	genericError = "genericError",
-
-	/**
-	 * Throttling error from server. Server is busy and is asking not to reconnect for some time
-	 */
-	throttlingError = "throttlingError",
-
-	/**
-	 * Data loss error detected by Container / DeltaManager. Likely points to storage issue.
-	 */
-	dataCorruptionError = "dataCorruptionError",
-
-	/**
-	 * Error encountered when processing an operation. May correlate with data corruption.
-	 */
-	dataProcessingError = "dataProcessingError",
+export { IErrorBase } from "@fluidframework/core-interfaces";
```

Comment:
Nit: you may want to move these deprecated re-exports to the package's index module - that way we can be sure this package isn't continuing to import from this module.

Reply:
Moved. Should I still keep the deprecated docs in the index file? https://github.com/microsoft/FluidFramework/pull/16868/commits/60ffe721be4d193e68a3367094426b585391fc02

Reply:
Yup!